<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ip-172-31-22-48" ignored="0" name="Test" tests="2" failures="1" timestamp="2022-09-22T13:26:49 UTC" time="3.411" errors="0">
  <testcase name="Test1" time="0.358" classname="testcases.LoginTest"/>
  <testcase name="Test2" time="0.479" classname="testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [Epic sadface: Username and password do not match any user in this service] but found [Epic sadface: Username is required]">
      <![CDATA[java.lang.AssertionError: expected [Epic sadface: Username and password do not match any user in this service] but found [Epic sadface: Username is required]
at testcases.LoginTest.Test2(LoginTest.java:39)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- Test -->
